### ISSUE TITLE
iOS Telegram Client with Tabs

### ISSUE THREAD

**Issue by uira** [R4dar/iOS#1]
## Problem
We want to deliver a iOS Client for Telegram API, that has tabs.

The example is the [Plus Messenger](https://github.com/R4dar/Plus-Messenger), that is basically an [Android only](https://play.google.com/store/apps/details?id=org.telegram.plus) Telegram with tabs.

![image](https://user-images.githubusercontent.com/89998/42666706-1a0eecb2-861d-11e8-986f-f294a57f98e6.png)

## Solution

Fork the [official Telegram iOS Client](https://github.com/peter-iakovlev/Telegram), implement the tabs  and send us the PR 🥇

**Comment by mijo-gracanin** [comment]
So you want to replace native iOS tab bar with custom (Android style) tab bar?

**Comment by uira** [comment]
In fact, the best solution is to implement this tabs in the traditional telegram iOS bar, including new options. is this?

![image](https://user-images.githubusercontent.com/89998/43822202-d289ba0c-9ac1-11e8-95e0-5d1091896886.png)

**Comment by mijo-gracanin** [comment]
I agree that it's the best to use native tab bar with your buttons. Keep in mind that, if you put more than 5 buttons in native tab bar, then fifth button becomes "More" button and opens context menu with the rest of buttons.

**Comment by uira** [comment]
Ok about 'More'. 

About this, the Plus Messenger has a simple "Sort Tabs" option, where users can configure. 

This can be implemented?

![photo_2018-08-08_16-10-54](https://user-images.githubusercontent.com/89998/43859045-032f10bc-9b26-11e8-8b80-6040fd5d2ff5.jpeg)

![image](https://user-images.githubusercontent.com/89998/43859055-05d77598-9b26-11e8-8d2c-e5b31a1795e1.png)

**Comment by mijo-gracanin** [comment]
Yeah, that can be easily implemented.

**Comment by uira** [comment]
Lets rockit!?

**Comment by mijo-gracanin** [comment]
I can allocate some time next week to make an example of configurable tab bar. We can discuss it via Telegram if you want: t.me/gmijo

**Comment by dlt5** [comment]
There are several clients with tabs for iOS. The best is Mobigram (ex-Mobogram iOS). Also iTele (beta is on new source code) and others.

**Comment by c0d3x27** [comment]
let me see if i understand... you want an app just like this one in the first pic that is for android but for IOS?

**Comment by c0d3x27** [comment]
> ## Problem
> 
> We want to deliver a iOS Client for Telegram API, that has tabs.
> 
> The example is the [Plus Messenger](https://github.com/R4dar/Plus-Messenger), that is basically an [Android only](https://play.google.com/store/apps/details?id=org.telegram.plus) Telegram with tabs.
> 
> ![image](https://user-images.githubusercontent.com/89998/42666706-1a0eecb2-861d-11e8-986f-f294a57f98e6.png)
> ## Solution
> 
> Fork the [official Telegram iOS Client](https://github.com/peter-iakovlev/Telegram), implement the tabs and send us the PR 🥇

let me see if i understand... you want an app just like this one in the first pic that is for android but for IOS?

**Comment by uira** [comment]
Yes
